### DISCUSSION TITLE
[YES] Discussion Title Structure?

### DISCUSSION THREAD

**Discussion by lostdesign** [storybookjs/storybook#11386] [General]
Would it maybe make sense to rename the titles if possible? Perhaps even follow some sort of structure?

[VUE] How to ...? | [TS] main.js config issues ...

There are quite a few discussions which just have `storybook` which doesn't make it easy to know if you can actually help in that area or not.

Can the type of discussion be changed by maintainers here afterwards as well? So that questions go to help?

**Comment by shilman** [comment]
Yes it can 👆 😆 

Great idea!!

**Reply by lostdesign** [reply]
Cool, while there aren't too many as of now, it'd be great to get into structure before its too late I guess? I could see discussions here being a good source for a FAQ or maybe even short tutorials. Having some kinda tag structure / proper titles will help that :D
